src/Microsoft.Health.Dicom.Core/Features/Common/IndexedFileProperties.cs

```diff
@@ -3,12 +3,15 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
+using System.Diagnostics.CodeAnalysis;
+
 namespace Microsoft.Health.Dicom.Core.Features.Common;
 
 /// <summary>
 /// Metadata on FileProperty table in database
 /// </summary>
-public class IndexedFileProperties
+[SuppressMessage("Performance", "CA1815:Override equals and operator equals on value types", Justification = "Identifiers are not equatable.")]
```

Comment:
For the sake of safety of other users (in case they try to use `==` in a test), I'd implement `Equals` and `GetHashCode` which should be pretty small here